server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -132,26 +137,25 @@ For more detail about how caches can be configured see <<tuningcache>>.
 
-The Infinispan caches on the {project_name} side must be configured with the link:https://infinispan.org/docs/8.2.x/user_guide/user_guide.html#remote_store[remoteStore] to ensure that data are saved to the remote cache. There is separate Infinispan cluster between JDG servers, so the data saved on JDG1 on `site1` are replicated to JDG2 on `site2` .
+The Infinispan caches on the {project_name} side must be configured with the link:https://infinispan.org/docs/8.2.x/user_guide/user_guide.html#remote_store[remoteStore] to ensure that data are saved to the remote cache. There is separate Infinispan cluster between {jdgserver_name} servers, so the data saved on JDG1 on `site1` are replicated to JDG2 on `site2` .
 
-Finally, the receiving JDG server notifies the {project_name} servers in its cluster through the Client Listeners, which are a feature of the HotRod protocol. {project_name} nodes on `site2` then update their Infinispan caches and the particular user session is also visible on {project_name} nodes on `site2`.
+Finally, the receiving {jdgserver_name} server notifies the {project_name} servers in its cluster through the Client Listeners, which are a feature of the HotRod protocol. {project_name} nodes on `site2` then update their Infinispan caches and the particular user session is also visible on {project_name} nodes on `site2`.
 
 See the <<archdiagram>> for more details.
 
-
 [[setup]]
-==== Basic setup
+==== Setting up Cross DC with {jdgserver_name} {jdgserver_version}
 
-For this example, we describe using two data centers, `site1` and `site2`. Each data center consists of 1 {jdgserver_name} server and 2 {project_name} servers. We will end up with 2 {jdgserver_name} servers and 4 {project_name} servers in total.
+This example for {jdgserver_name} {jdgserver_version} involves two data centers, `site1` and `site2`. Each data center consists of 1 {jdgserver_name} server and 2 {project_name} servers. We will end up with 2 {jdgserver_name} servers and 4 {project_name} servers in total.
 
 * `Site1` consists of {jdgserver_name} server, `jdg1`, and 2 {project_name} servers, `node11` and `node12` .
 
 * `Site2` consists of {jdgserver_name} server, `jdg2`, and 2 {project_name} servers, `node21` and `node22` .
 
 * {jdgserver_name} servers `jdg1` and `jdg2` are connected to each other through the RELAY2 protocol and `backup` based {jdgserver_name}
-caches in a similar way as described in the link:{jdgserver_crossdcdocs_link}[JDG documentation].
+caches in a similar way as described in the link:{jdgserver_crossdcdocs_link}[{jdgserver_name} documentation].
 
 * {project_name} servers `node11` and `node12` form a cluster with each other, but they do not communicate directly with any server in `site2`.
 They communicate with the Infinispan server `jdg1` using the HotRod protocol (Remote cache). See <<communication>> for the details.
```

Comment:
line 163 in this doc, typo:
`HotRod` => `Hot Rod`
You may want to do a find&replace in the doc.

Reply:
Line 174 and below:
This doc needs to be updated to Infinispan 11 / DG8. It mentions the old server configuration, example `JDG1_HOME/standalone/configuration/clustered.xml` is no longer valid.

Reply:
Good point. It needs to be likely RHDG_HOME/server/conf/infinispan-xsite.xml as mentioned in the instructions https://issues.redhat.com/secure/attachment/12502085/HOW-TO-rhdg81-rhsso744.txt .

Reply:
I think these instructions are still valid for DG 7.3. Not sure but I believe it is the case that SSO supports both 7.3 and 8.1. So this probably should stay as-is.